src/electron/windows/sign_windows_executable.action.mjs

```diff
@@ -192,7 +161,7 @@ async function signWindowsExecutable(exeFile, algorithm, options) {
   try {
     exitCode = await jsign(exeFile, jsignArgs);
   } catch (err) {
-    console.error('failed to start java, please make sure you have installed java');
+    console.error('failed to download and run jsign, please also make sure java is installed');
```

Comment:
Say it failed to run jsign, and output the error that was thrown, so we know the cause.

Reply:
We don't need to show the error here because next line I `throw` the error and the action runner will display that in the console. I can simplify the error message to `failed to run jsign`.